.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug report
+description: Encountered a clear bug or crash with logs backing it? Fill it up!
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+
+        And also search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
+
+  - type: dropdown
+    attributes:
+      label: Type
+      description: |
+          - Game behaviour bug: game gets into an unexpected state (e.g. incorrect scoring, wrong multiplayer room state, etc.).
+          - Cosmetic bug: something looks wrong (e.g. misaligned component, text not wrapping, etc.)
```

Comment:
Remove these, I didn't intend for it to be verbatim copied in here.

Reply:
Looked good for explaining the difference between `Game behaviour` and `Cosmetic`, but sure.